Sources/BottomSheetView.swift

```diff
@@ -115,7 +125,6 @@ public final class BottomSheetView: UIView {
         superview.layoutIfNeeded()
         addGestureRecognizer(panGesture)
 
-        currentTargetOffsetIndex = targetIndex
```

Comment:
Don't we still need this?

Reply:
Kind of, but we can't use `targetIndex` directly since we are sorting `targetOffsets`.

Reply:
@vadymmarkov This should be fixed now